go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 // CAVEAT: This introduces a circular dependency. If you change this line, you MUST change the "exclude"
 // directive at the bottom of the file as well.
-require github.com/stackrox/scanner v0.0.0-20220426001230-9ab6777c9581
+require github.com/stackrox/scanner v0.0.0-20220929215850-334d4658d5e6
```

Comment:
It looks like the above comment is outdated, since you didn't actually update the `exclude` directive. Remove the comment and the `exclude` directive